.ci/spec/run-test.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+csource="${BASH_SOURCE[0]}"
+while [ -h "$csource" ] ; do csource="$(readlink "$csource")"; done
+root="$( cd -P "$( dirname "$csource" )/../" && pwd )"
+
+. "${root}/.ci/spec/env.sh"
+. "${root}/.ci/common.sh"
+
+pushd "${root}" > "${output}"
+
+# Assumes packet is already set up
+
+# Define a custom cleanup function to destroy any orphan guests
+# on the packet device
+function cleanup() {
+    (>&2 echo "Cleaning up packet device")
+    unset PACKET_EXEC_PERSIST
+    pkt_wrap_stream "cd vagrant;vagrant destroy -f" \
+                "Vagrant command failed"
+}
+
+# job_id is provided by common.sh
+export PACKET_EXEC_REMOTE_DIRECTORY="${job_id}"
+
+# spec test configuration, defined by action runners, used by Vagrant on packet
+export PKT_VAGRANT_HOST_BOXES="${VAGRANT_HOST_BOXES}"
+export PKT_VAGRANT_GUEST_BOXES="${VAGRANT_GUEST_BOXES}"
+# other vagrant-spec options
+export PKT_VAGRANT_HOST_MEMORY=10000
```

Comment:
We shouldn't need the host memory value here.

Reply:
We have to here for Windows tests, otherwise there won't be enough memory to run them.